engine/src/main/java/org/terasology/rendering/particles/internal/rendering/DisplayListParticleRenderer.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2015 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.particles.internal.rendering;
+
+import org.lwjgl.opengl.GL11;
+import org.lwjgl.opengl.GL13;
+import org.terasology.math.geom.Vector3f;
+import org.terasology.rendering.assets.material.Material;
+import org.terasology.rendering.particles.components.ParticleSystemComponent;
+import org.terasology.rendering.particles.internal.data.ParticlePool;
+import org.terasology.rendering.particles.internal.data.ParticleSystemStateData;
+
+import static org.lwjgl.opengl.GL11.*;
```

Comment:
Would this be caught by IntelliJ's format code/optimize imports on PRs? I've checked the boxes for my PRs all the time, I think this will apply to all modified files in the PR...